lib/main.dart

```diff
@@ -28,6 +31,13 @@ void main () {
     SystemChrome.setSystemUIOverlayStyle(const SystemUiOverlayStyle(
       statusBarColor: Colors.transparent, // transparent status bar
     ));
+
+    WidgetsFlutterBinding.ensureInitialized();
+    if (Platform.isWindows || Platform.isLinux || Platform.isMacOS) {
+      setWindowTitle('Source Server Manager');
+      setWindowMinSize(const Size(600, 900));
+      setWindowMaxSize(const Size(700, 900));

```

Comment:
UI not scalable so just fixing it for now, but after releasing on windows store, creating a scalable UI is definitely on the plate.